modules/installation-guide/partials/proc_configuring-labels-for-routes-ingresses.adoc

```diff
@@ -0,0 +1,40 @@
+
+[id="configuring-labels-for-routes_ingresses_{context}"]
+= Configuring labels for routes or ingresses
+
+This procedure describes how to configure labels for routes or ingresses to organize and categorize (scope and select) objects.
+
+.Prerequisites
+
+* The `{orch-cli}` tool is available.
+* {prod-short} is deployed in a Kubernetes cluster.
+
+IMPORTANT: It is needed to use comma to separate labels: `key1=value1,key2=value2`
+
+.Procedure
+
+. To configure labels for routes update the Custom Resource with the following commands:
++
+[subs="+quotes,+attributes"]
+----
+$ {orch-cli} patch checluster {prod-checluster} -n {prod-namespace} --type=json -p '[{"op": "replace", "path": "/spec/server/cheServerRoute/labels", "value": "__<labels for a {prod-id-short} server route>__"}]'
```

Comment:
Is it possible to add line breaks without breaking the commands? I started this as an example, but I am not sure how to break lines inside the JSON without breaking it:

![Screenshot from 2020-09-28 16-30-12](https://user-images.githubusercontent.com/243761/94445637-f49c9200-01a7-11eb-83d3-a5224b278271.png)

Reply:
done
![tmp](https://user-images.githubusercontent.com/1640675/94524963-02035c00-023c-11eb-88ec-4ce10df25146.png)